.github/workflows/build-container.yml

```
@@ -6,6 +6,9 @@ on:
       - master
   schedule:
     - cron: '7 0 * * sun'
+  repository_dispatch:
+    types: [dispatch-build]
+  workflow_dispatch:
```

Is yaml ok with an empty list/dict like this? Maybe you need
```suggestion
  workflow_dispatch: {}
```
?

---

Yeah, surprisingly it is. I first noticed it in https://github.com/opensciencegrid/osg-gfactory/blob/master/OSG_autoconf/10-hosted-ces.auto.yml#L113-L116

---

OK but I think those end up as empty strings whereas workflow_dispatch is supposed to be a dict.